vite.config.js

```diff
@@ -16,6 +16,11 @@ export default defineConfig({
   build: {
     sourcemap: true
   },
+  template: {
+    compilerOptions: {
+      isCustomElement: (tag) => ['model-viewer'].includes(tag),
+    }
+  },
```

Comment:
Why these new lines? 

The `template` is already configured before in the `plugins` settings.

Reply:
It showed some warning. I removed it via a new commit.